<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1308</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>All examiners</string>
    </property>
   </widget>
   <widget class="QListView" name="allExaminersList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>471</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="findExaminerList">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>471</width>
      <height>641</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Find examiner</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="findSurnameInput">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Surname</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findExaminerButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Find</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportDocx">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>640</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Export info about all examiners in docx files</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportXlsx">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>670</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Export info about all examiners in XLSX file</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add examiner</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="surnameEdit">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>40</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Surname</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>80</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="patronymicEdit">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>120</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Patronymic</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addExaminerButton">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>210</y>
      <width>107</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add examiner</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="salaryEdit">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>170</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>500000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>170</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Salary:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="examinerIDInput">
    <property name="geometry">
     <rect>
      <x>1171</x>
      <y>420</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>310</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>_____________________________________________________</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>420</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Examiner ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>270</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>_____________________________________________________</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>250</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>_____________________________________________________</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>370</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Manipulate examiner</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>330</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>_____________________________________________________</string>
    </property>
   </widget>
   <widget class="QPushButton" name="manipulateExaminerButton">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>460</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Manipulate examiner</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>290</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>_____________________________________________________</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateAllListBtn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Update list</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1308</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEnrollee">
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="title">
     <string>Go to</string>
    </property>
    <addaction name="actionEnrollee"/>
    <addaction name="actionExam"/>
   </widget>
   <addaction name="menuEnrollee"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEnrollee">
   <property name="text">
    <string>Enrollee</string>
   </property>
  </action>
  <action name="actionExaminer">
   <property name="text">
    <string>Examiner</string>
   </property>
  </action>
  <action name="actionExam">
   <property name="text">
    <string>Exam</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
